### PR TITLE
config fixes

### PR DESCRIPTION
**Pull request by danbornside**
miscellaneous yak shaves to get external javascript dependencies to work.
 - merged in the overridable closure compiler level
 - adjustments to the new ExecutableConfig Api to make it actually work.

### CODE REVIEW

File: lib/run/src/Obelisk/Run.hs [hunk 1]
```diff
@@ -95,7 +95,7 @@ runServeAsset :: FilePath -> [Text] -> Snap ()
 runServeAsset rootPath = Snap.serveAsset "" rootPath . T.unpack . T.intercalate "/"
 
 getConfigRoute :: Map Text Text -> IO (Maybe URI)
-getConfigRoute configs = case Map.lookup "common/route" configs of
+getConfigRoute configs = case Map.lookup "config/common/route" configs of
```
Comment (cgibbard) [inline lib/run/src/Obelisk/Run.hs]: this change is interesting -- somehow the configs being passed here include stuff outside the config directory now?

Comment (ryantrinkle) [reply lib/run/src/Obelisk/Run.hs]: No, this was just a poorly-conceived change

Comment (ryantrinkle) [reply lib/run/src/Obelisk/Run.hs]: (the one that made this change necessary, i mean)

**Comment by ali-abrar** [comment]
Superceded by #452

**Pull request closed without merging**